integration-tests/tests/utils/aws/dynamoDB.ts

```diff
@@ -130,60 +53,95 @@ function getFieldListValues(ticketDetails: ItemDetails, fieldID: number) {
   }
 }
 
-function getFieldValue(ticketDetails: ItemDetails, fieldID: number) {
+const getFieldValue = (ticketDetails: ItemDetails, fieldID: number) => {
   const field = ticketDetails.fields.filter((field) => {
     return field.id === fieldID
   })
   return field.pop()?.value
 }
 
-export const getValueFromDynamoDB = async (
+const generateDynamoTableEntry = (
   ticketId: string,
-  attributeName?: string
+  ticketDetails: ItemDetails
 ) => {
-  let getAttributeValueParams
-  if (attributeName) {
-    getAttributeValueParams = {
-      TableName: AUDIT_REQUEST_DYNAMODB,
-      Key: {
-        zendeskId: { S: `${ticketId}` }
-      },
-      ProjectionExpression: attributeName
-    }
-  } else {
-    getAttributeValueParams = {
-      TableName: AUDIT_REQUEST_DYNAMODB,
-      Key: {
-        zendeskId: { S: `${ticketId}` }
+  return {
```

Comment:
If we're just returning this object we could use paranthesis and drop the return statement

Reply:
thanks! That's done